docker/buildwheel.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
+DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$DOCKER_DIR/shared.env"
+
 set -e -x
 
 ARCH=`uname -p`
 echo "arch=$ARCH"
 
-for V in cp37-cp37m cp36-cp36m cp35-cp35m cp27-cp27m cp27-cp27mu; do
+for V in "${PYTHON_VERSIONS[@]}"; do
```

Comment:
Is there any reason this isn't (and the same for runtests.sh):

```suggestion
for PYBIN in /opt/python/*/bin; do
```

Reply:
The image ships with lots of unsupported Python versions.